### PR TITLE
Fixes *some* robotics consoles being unable to be deconstructed

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[BUG]
## About the PR 
Did you know that the robotics consoles specifically on Kondaru and Donut 2 were impossible to deconstruct?
Yeah not anymore, let's be at least slightly consistent.

## Why's this needed? 
Consistency between maps 🙏 

## Changelog

```changelog
(u)LeahTheTech
(+)Robotics consoles on Kondaru and Donut 2 have had their safety screws removed due to budget cuts.
```

### CODE REVIEW

**Comment by frawhst** [comment]
Didn’t we fix this like 6 months ago
oh donut 2 nvm

**Comment by TobleroneSwordfish** [comment]
Apparently Kondaru too? 🤷‍♀️

File: maps/ozymandias.dmm [hunk 1]
```diff
@@ -74736,10 +74736,6 @@
 	icon_state = "crewquarters";
 	name = "Warrens Hall"
 	})
-"thF" = (
-/obj/machinery/hydro_mister,
-/turf/simulated/floor/plating,
-/area/station/maintenance/inner/hydroponics)
```
Comment (ZeWaka) [inline maps/ozymandias.dmm:74742]: ?

Comment (TobleroneSwordfish) [reply maps/ozymandias.dmm:74742]: uhhhhh

**Comment by TobleroneSwordfish** [comment]
time to remake branch :)))))))))

**Pull request closed without merging**